packages/orbit-components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -78,110 +81,80 @@ export const WithTextLinkInLabel = ({ checked }) => {
           .
         </Text>
       }
-      checked={checked}
-      value="value"
-      onChange={action("changed")}
     />
-  );
-};
-
-WithTextLinkInLabel.story = {
-  name: "With TextLink in label",
+  ),
 
   parameters: {
-    info: "Additionally you can add info to this component.",
+    info: "Additionally you can info text and link to this component.",
+    controls: {
+      exclude: ["onChange", "label"],
+    },
   },
-};
 
-WithTextLinkInLabel.args = {
-  checked: true,
+  args: {
+    checked: true,
+  },
 };
 
-export const WithTooltip = () => {
-  return (
-    <Checkbox
-      label="Direct"
-      value="one"
-      disabled
-      tooltip={
-        <Tooltip content="There are no results available with this option" placement="top" />
-      }
-    />
-  );
-};
+export const WithTooltip: Story = {
+  render: args => {
+    return (
+      <Checkbox
+        {...args}
+        tooltip={
+          <Tooltip content="There are no results available with this option" placement="top" />
+        }
+      />
+    );
+  },
+
+  args: {
+    disabled: true,
+  },
 
-WithTooltip.story = {
   parameters: {
-    info: "Additionally you can add tooltip to this component.",
+    controls: {
+      exclude: ["onChange"],
+    },
   },
 };
 
-export const Playground = ({
-  label,
-  value,
-  checked,
-  disabled,
-  hasError,
-  readOnly,
-  info,
-  dataTest,
-  name,
-}) => {
-  return (
-    <Checkbox
-      label={label}
-      value={value}
-      checked={checked}
-      disabled={disabled}
-      name={name}
-      hasError={hasError}
-      dataTest={dataTest}
-      info={info}
-      readOnly={readOnly}
-      onChange={action("changed")}
-    />
-  );
-};
+export const ReadOnly: Story = {
+  args: {
+    readOnly: true,
+  },
 
-Playground.story = {
   parameters: {
-    info: "Playground of Checkbox",
+    info: "This is a preview of this component in read-only state.",
+    controls: {
+      exclude: ["onChange"],
+    },
   },
 };
 
-Playground.args = {
-  label: "Label",
-  value: "value",
-  checked: false,
-  disabled: false,
-  hasError: false,
-  readOnly: false,
-  info: "Additional information for this choice",
-  dataTest: "test",
-  name: "name",
-};
-
-export const Rtl = () => (
-  <RenderInRtl>
-    <Checkbox
-      label={
-        <Text>
-          I instruct Kiwi.com to cancel this booking under the herein specified conditions and to
-          process a refund in accordance with Kiwi.com&rsquo;&nbsp;
-          <TextLink>Terms and Conditions</TextLink>.
-        </Text>
-      }
-      checked
-      value="value"
-      onChange={action("changed")}
-    />
-  </RenderInRtl>
-);
-
-Rtl.story = {
-  name: "RTL",
+export const Rtl: Story = {
+  render: args => (
+    <RenderInRtl>
+      <Checkbox
+        {...args}
+        label={
```

Comment:
Hm, is the same here with label as I previously mentioned in different story. Maybe let's disable it also here?

Reply:
good point. 👍